operator/pkg/manifest/installer.go

```diff
@@ -968,3 +968,35 @@ func logAndPrint(v ...interface{}) {
 	scope.Infof(s)
 	fmt.Println(s)
 }
+
+// AddManagedLabelsNoReorder applies two label rules to the given manifest map:
+// 1. Adds metadata.labels to any resource without this key.
+// 2. Add metadata.labels.operatorLabelStr: operatorReconcileStr label, which marks the resource as being managed
+// by istioctl. This is used to select resources for pruning.
+func AddManagedLabelsNoReorder(mm name.ManifestMap) (name.ManifestMap, error) {
+	rules := []*util.Rule{
+		{
+			Path: util.PathFromString("metadata"),
+			Key:  "labels",
+		},
+		{
+			Path: util.PathFromString("metadata.labels"),
+			Key:  operatorLabelStr,
+			Val:  operatorReconcileStr,
+		},
```

Comment:
Do we have plan to align the labels with in-cluster owner labels?

Reply:
Not really - this is just to help those folks that really wish to use manifest generate to install (which we don't recommend). There's no plan to make this equivalent to using apply.